### PR TITLE
FEATURE : Add sidebar and loader animation

### PR DESCRIPTION
**Pull request by sahsisunny**
### This PR is part of the #371 

**Feature added:**
- Sidebar panel
- Loader

https://github.com/Real-Dev-Squad/website-dashboard/assets/70854507/61df8125-f82b-44aa-b798-f128542bf1ce

#### Test  - #375

### CODE REVIEW

**Review by RitikJaiswal75 — COMMENTED** [review]
Are there no tests for the js written? if no why?

**Comment by sahsisunny** [comment]
> Are there no tests for the js written? if no why?

Done

**Review by Pratiyushkumar — APPROVED** [review]
LGTM

File: standup/style.css [hunk 1]
```diff
@@ -216,6 +225,99 @@ tr:hover {
...
+  position: fixed;
+  z-index: 999;
+  top: 0;
+  right: 0;
+  color: var(--white-color);
+  overflow-x: hidden;
+  padding-left: 1rem;
+  padding-top: 6rem;
+  border-radius: 1rem 0 0 1rem;
+  background: rgba(14, 12, 168, 0.691);
```
[the remark this answers is not in the document]
Comment (sahsisunny) [reply standup/style.css]: Done

**Review by vinit717 — APPROVED** [review]

**Review by Pratiyushkumar — APPROVED** [review]
LGTM

**Pull request merged**